src/NuGet.Clients/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -56,6 +56,9 @@ public class RestoreCommand : DownloadCommandBase
         [Option(typeof(NuGetCommand), "ForceRestoreCommand")]
         public bool Force { get; set; }
 
+        [Option(typeof(NuGetCommand), "RestoreCommandSkipMSBuild")]
+        public bool SkipMSBuild { get; set; }
```

Comment:
Naming this `PackagesConfigOnly` would be more clear in my opinion.

Without reading the description of the command it isn't obvious what role MSBuild plays in restore, and if other commands such as Pack which requires MSBuild would break later if restore was called with SkipMSBuild.

Reply:
Good point. Unless someone else objects I'll make that change.

Reply:
Done.